priority_queue.py

```diff
@@ -16,12 +17,10 @@ def insert(self, item, priority=None):
         behavior through pop() and insert() functions.
         Priority levels are abritrarily set to only 'high' or 'low'.
         """
-        if priority == "high":
-            self.high.enqueue(item)
-        elif priority == "low":
-            self.low.enqueue(item)
-        else:
-            raise ValueError("Priorty must be literal 'high' or 'low'.")
+        try:
+            self.p[priority].enqueue(item)
+        except ValueError:
+            return "the priority value must be in the range 0 to " + str(self.max)
```

Comment:
Where exactly does `self.max` come from?  I'm not seeing it anywhere in the definition of this method, or in the definition of the class.  Did you run tests before submitting this?